frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -397,13 +403,20 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
               {!validAllowedBusses.has(bus) && (
                 <FormSelectOption
+                  isDisabled
                   key={bus.getValue()}
                   value={bus.getValue()}
-                  label={`${bus.toString()} --- Invalid ---`}
+                  label={bus.toString()}
                 />
               )}
               {[...validAllowedBusses].map((b) => (
-                <FormSelectOption key={b.getValue()} value={b.getValue()} label={b.toString()} />
+                <FormSelectOption
+                  key={b.getValue()}
+                  value={b.getValue()}
+                  label={`${b.toString()} ${
```

Comment:
space has to be conditional as well

Reply:
maybe we could remove this not recommended flag altogether?  as the same thing is shown with busValidation

Reply:
or at least remove it once it is selected?